<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configure</class>
 <widget class="QDialog" name="Configure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>318</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configure</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/qdacco.png</normaloff>:/icons/qdacco.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="ldiccionari">
         <property name="text">
          <string>Default dictionary:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>traduccio_defecte</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="traduccio_defecte">
         <item>
          <property name="text">
           <string>Catalan → English</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>English → Catalan</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lidioma">
         <property name="text">
          <string>User interface language:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>idioma_defecte</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="idioma_defecte">
         <item>
          <property name="text">
           <string>Catalan</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>English</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="laccents">
         <property name="text">
          <string>Ignore diacritics:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>ignorar_accents</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="ignorar_accents">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lmajuscules">
         <property name="text">
          <string>Match case:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>ignorar_majuscules</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="ignorar_majuscules">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="ldirectori">
         <property name="text">
          <string>Dictionary directory:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>directori</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="directori">
         <property name="toolTip">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="busca_fitxer">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/fileopen.png</normaloff>:/icons/fileopen.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="ajuda">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/help.png</normaloff>:/icons/help.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbrowser">
         <property name="text">
          <string>Browser:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>browser</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="browser">
         <property name="toolTip">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QToolButton" name="busca_browser">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/fileopen.png</normaloff>:/icons/fileopen.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QToolButton" name="ajuda_browser">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/help.png</normaloff>:/icons/help.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lfestival">
         <property name="text">
          <string>Festival (speech synthesis):</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="buddy">
          <cstring>festival</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="festival"/>
       </item>
       <item row="2" column="2">
        <widget class="QToolButton" name="busca_festival">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/fileopen.png</normaloff>:/icons/fileopen.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QToolButton" name="ajuda_festival">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/help.png</normaloff>:/icons/help.png</iconset>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Autocomplete:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>autocompletar</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="autocompletar">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Single instance:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>singleinstance</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="singleinstance">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QToolButton" name="ajuda_singleinstance">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/help.png</normaloff>:/icons/help.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="okCancel">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>traduccio_defecte</tabstop>
  <tabstop>idioma_defecte</tabstop>
  <tabstop>ignorar_accents</tabstop>
  <tabstop>ignorar_majuscules</tabstop>
  <tabstop>directori</tabstop>
  <tabstop>busca_fitxer</tabstop>
  <tabstop>ajuda</tabstop>
  <tabstop>browser</tabstop>
  <tabstop>busca_browser</tabstop>
  <tabstop>ajuda_browser</tabstop>
  <tabstop>festival</tabstop>
  <tabstop>busca_festival</tabstop>
  <tabstop>ajuda_festival</tabstop>
  <tabstop>autocompletar</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
